docs/development/classlib/classlib-manual.rst

```diff
@@ -0,0 +1,72 @@
+.NET Core Class Library Manual
```

Comment:
So in a sense you're asking what the targeting story for .NET Framework looks like. I don't expect that to change. If you want to target .NET Framework, you'll need to create a .NET Framework specific class library which means you're aren't portable to anything. Or you can use PCL/.NET Core which means can also run on .NET Framework, however, you'll not be able to use functionality that isn't in .NET Core.

Personally, I expect the long term path for .NET Framework to be cross-compilation and within .NET Core ecosystem to be like PCLs (i.e. based on binary sharing).